pkg/odf/dr/dr.go

```diff
@@ -0,0 +1,98 @@
+package dr
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	"github.com/red-hat-storage/odf-cli/cmd/odf/root"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+	submarinerv1alpha1 "github.com/submariner-io/submariner-operator/api/v1alpha1"
+	submarinerv1 "github.com/submariner-io/submariner/pkg/apis/submariner.io/v1"
+	"github.com/submariner-io/submariner/pkg/cidr"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const submarinerOperatorNamespace string = "submariner-operator"
```

Comment:
The submariner operator is always in this ns?

Reply:
Yes

Reply:
do we want to cli option for this to overwrite?

Reply:
I don't see the need as this is hardcoded within submariner project as well.
We can assess it when the need arises.